### PR TITLE
Safety check when asking for new spsa parameters.

### PR DESCRIPTION
**Pull request by vdbergh**
It is possible that the task is finished (not tested).

### CODE REVIEW

File: worker/games.py [hunk 1]
```diff
@@ -1117,7 +1122,7 @@ def run_games(worker_info, password, remote, run, task_id, pgn_file):
     if "start" in task:
         print("Variable task sizes used. Opening offset = {}".format(opening_offset))
     start_game_index = opening_offset + input_total_games
-    run_seed = int(hashlib.sha1(run["_id"].encode("utf-8")).hexdigest(), 16) % (2**30)
+    run_seed = int(hashlib.sha1(run["_id"].encode("utf-8")).hexdigest(), 16) % (2 ** 30)
```
Comment (ppigazzini) [inline worker/games.py:1125]: The last version of `black` switched from `x ** y` format to `x**y`.
Just as information, I usually auto format in the worker version raising commit.

**Comment by vdbergh** [comment]
I tested it on DEV by stopping an SPSA test while the worker was still compiling. The output was as expected.
```
Info from remote: Request_spsa: task 620635f5d71106ed12a46cd0/294 is not active
Post request https://dfts-0.pigazzini.it:443/api/request_spsa handled in 108.94ms (server: 1.99ms)
Server told us task is no longer needed
Task exited
```

**Review by ppigazzini — APPROVED** [review]
Tested working also with windows worker.

**Comment by ppigazzini** [comment]
Triggered the workers update, thank you @vdbergh :)

**Pull request merged**